chat.js

```diff
@@ -13,16 +13,28 @@ const io = require("socket.io")(server, {
 });
 const port = 4000;
 const cors = require("cors");
-
+const clients = [];
```

Comment:
could this cause issues in future rooms? possibly add a feature for multiple rooms?